Build failure: cemu

### Steps To Reproduce

Steps to reproduce the behavior:

1. Use the following `flake.nix`:

```nix
{
  inputs.nixpkgs.url = "github:nixpkgs/nixos/c002c6aa977ad22c60398daaa9be52f2203d0006";

  outputs =
    { self, nixpkgs }:
    {
      packages.x86_64-linux.default = nixpkgs.legacyPackages.x86_64-linux.cemu;
    };
}

```

2. Build the flake:
```sh
$ nix build
```

### Build log

```
error: builder for '/nix/store/xv95p8sdyj9yxb123mgqd2hdapvrblw3-cemu-2.0-65.drv' failed with exit code 2;
       last 10 log lines:
       > [ 56%] Building CXX object src/Cafe/CMakeFiles/CemuCafe.dir/HW/Latte/Renderer/Vulkan/VulkanAPI.cpp.o
       > [ 56%] Building CXX object src/Cafe/CMakeFiles/CemuCafe.dir/HW/Latte/Renderer/Vulkan/VulkanPipelineCompiler.cpp.o
       > /build/source/src/Cafe/HW/Latte/Renderer/Vulkan/VKRPipelineInfo.cpp:6: fatal error: glslang/Include/Types.h: No such file or directory
       >     6 | #include <glslang/Include/Types.h>
       >       |
       > compilation terminated.
       > make[2]: *** [src/Cafe/CMakeFiles/CemuCafe.dir/build.make:1725: src/Cafe/CMakeFiles/CemuCafe.dir/HW/Latte/Renderer/Vulkan/VKRPipelineInfo.cpp.o] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[1]: *** [CMakeFiles/Makefile2:547: src/Cafe/CMakeFiles/CemuCafe.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/xv95p8sdyj9yxb123mgqd2hdapvrblw3-cemu-2.0-65.drv'.
error: 1 dependencies of derivation '/nix/store/ddjpj3xz2cli4wf526b95a8ql1rnw9cb-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/cajvzwwpjxwrcp6fka3hyqqg5n1cs2xb-nixos-system-amaurot-24.05.20240127.c002c6a.drv' failed to build
```

### Notify maintainers

cc @zhaofengli @baduhai

### Metadata

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.1, NixOS, 24.05 (Uakari), 24.05.20240121.612f972`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/emulators/cemu/default.nix
+++ b/pkgs/applications/emulators/cemu/default.nix
@@ -1,4 +1,5 @@
 { lib, stdenv, fetchFromGitHub
+, fetchpatch
 , addOpenGLRunpath
 , wrapGAppsHook
 , cmake
@@ -47,6 +48,13 @@ stdenv.mkDerivation rec {
     # > The following imported targets are referenced, but are missing:
     # > SPIRV-Tools-opt
     ./cmakelists.patch
+
+    # Remove on next release
+    # https://github.com/cemu-project/Cemu/pull/1076
+    (fetchpatch {
+      url = "https://github.com/cemu-project/Cemu/commit/72aacbdcecc064ea7c3b158c433e4803496ac296.patch";
+      hash = "sha256-x+ZVqXgGRSv0VYwJAX35C1p7PnmCHS7iEO+4k8j0/ug=";
+    })
   ];
 
   nativeBuildInputs = [

